website/docs/r/sql_managed_instance_failover_group.html.markdown

```diff
@@ -0,0 +1,152 @@
+  administrator_login_password = "thisIsDog11"
+  license_type                 = "BasePrice"
+  subnet_id                    = azurerm_subnet.secondary.id
+  sku_name                     = "GP_Gen5"
+  vcores                       = 4
+  storage_size_in_gb           = 32
+
+  depends_on = [
+    azurerm_subnet_network_security_group_association.secondary,
+    azurerm_subnet_route_table_association.secondary,
+  ]
+
+  tags = {
+    environment = "prod"
+  }
+}
+
+resource "azurerm_sql_managed_instance_failover_group" "example" {
+  name                  = "example-failover-group"
+  resource_group_name   = azurerm_resource_group.primary.name
+  location              = azurerm_sql_managed_instance.primary.location
+  managed_instance_name = azurerm_sql_managed_instance.primary.name
+
+  partner_managed_instances {
+    id = azurerm_sql_managed_instance.secondary.id
+  }
+
+  read_write_endpoint_failover_policy {
+    mode          = "Automatic"
+    grace_minutes = 60
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this SQL Instance Failover Group. Changing this forces a new SQL Instance Failover Group to be created.
+
+* `managed_instance_name` - (Required) The name of the SQL Managed Instance which will be replicated using a SQL Instance Failover Group. Changing this forces a new SQL Instance Failover Group to be created.
+
+* `location` - The Azure Region where the SQL Instance Failover Group exists.
+
+* `partner_managed_instances` - (Required) One or more `partner_managed_instances` blocks as defined below.
+
+* `read_write_endpoint_failover_policy` - (Required) A `read_write_endpoint_failover_policy` block as defined below.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the SQL Instance Failover Group should exist. Changing this forces a new SQL Instance Failover Group to be created.
+
+---
+
+* `readonly_endpoint_failover_policy` - (Optional) A `readonly_endpoint_failover_policy` block as defined below.
+
+---
+
+A `partner_managed_instances` block supports the following:
+
+* `id` - (Required) ID of the SQL Managed Instance which will be replicated to.
+
+---
+
+A `read_write_endpoint_failover_policy` block supports the following:
+
+* `mode` - (Required) The failover mode. Possible values are `Manual`, `Automatic`
+
+* `grace_minutes` - (Optional) Applies only if `mode` is `Automatic`. The grace period in minutes before failover with data loss is attempted.
+
+---
+
+A `readonly_endpoint_failover_policy` block supports the following:
+
+* `mode` - (Required) Failover policy for the read-only endpoint. Possible values are `Enabled`, and `Disabled`
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the SQL Instance Failover Group.
+
+* `partner_regions` - A `partner_regions` block as defined below.
+
+* `role` - TODO.
```

Comment:
Are these `TODO`s something we need to look into?

Reply:
Fixed ✅